ui/README.md

```diff
@@ -6,30 +6,42 @@ This is the web UI for PeopleMover.
 ### Getting Started
 These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
 
-### Built with
-* [React](https://reactjs.org/)
-* [React-Redux](https://react-redux.js.org/)
-* [React Testing Library](https://github.com/testing-library/react-testing-library)
-* [TypeScript](https://www.typescriptlang.org/)
-* [Node.js](https://nodejs.org/en/)
+### Dependencies
+- [Node 14](https://nodejs.org)
 
 ### Build with NPM
 1. Run `npm install` to install the dependencies
 2. Build the project with the following command: `npm run build`
 
+## Testing
+
+### Unit Tests
+```
+npm run test
```

Comment:
This runs both the unit tests and the e2e tests. I think you want npm run test:unit here.